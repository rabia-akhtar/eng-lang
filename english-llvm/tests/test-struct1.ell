struct Book {
	int Id;
	string Title;
};

struct Doc {
	string File;
	int Word_count;
};

int main()
{
  struct Book b1;
  struct Doc doc;
  b1.Id = 2;
  b1.Title = "hello world";
  doc.File = "this is an essay";
  doc.Word_count = 20;
  print(b1.Id);
  print_string(b1.Title);
  print_string(doc.File);
  print(doc.Word_count);
  return 0;
}