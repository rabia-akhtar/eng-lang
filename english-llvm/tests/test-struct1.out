2
hello world
this is an essay
20